hhvm/deb/skeleton/DEBIAN/postinst

```diff
@@ -38,6 +38,12 @@ echo "*    --install /usr/bin/php php /usr/bin/hhvm 60"
 
 echo "********************************************************************"
 
+if [ ! -d "/var/run/hhvm" ]; then
+    mkdir -p /var/run/hhvm
+    chmod 644 /var/run/hhvm
+    chown www-data:www-data /var/run/hhvm
```

Comment:
644 is not good mode for directory! you did not seem to actually test your changes!

and why not use install(1) to make whole change in one command?

```
install -d -m 755 -o www-data -g www-data /var/run/hhvm
```